.github/workflows/integration.yml

```diff
@@ -0,0 +1,25 @@
+name: Compstore UI Continuous Integration
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  test_pull_request:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "18"
+
+      - name: Install Prettier
+        run: npm install prettier
+
+      - name: Code Formatting Prettier
+        run: npm run format
```

Comment:
Na front przed dodaniem do mastera zrobiłem tylko instalację prettier i formatowanie, bo nie są potrzebne nam inne bibioteki (node_modules waży już 564 MB, ale jak robi obraz dockerowy to ma tylko 45 MB). Src waży 237 KB.

Reply:
Zastanawiam się czy dałoby się ominąć to powtarzalne instalowanie prettier
Gdzieś rzuciło mi się w oczy coś takiego jak cachowanie
https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows
Lecz wydaje mi się to skomplikowane na teraz. Jeśli instalowanie będzie wyraźnie wpływać na czas wykonania to będziemy mogli zainteresować się tym cachowaniem